openedx_webhooks/tasks/github_work.py

```diff
@@ -12,12 +14,19 @@
 )
 
 
+def get_repo_labels(repo: str) -> Dict[str, Dict[str, Any]]:
+    """Get a dict mapping label names to full label info."""
+    url = f"/repos/{repo}/labels"
+    repo_labels = {lbl["name"]: lbl for lbl in paginated_get(url, session=get_github_session())}
+    return repo_labels
+
+
 @memoize_timed(minutes=15)
 def synchronize_labels(repo: str) -> None:
     """Ensure the labels in `repo` match the specs in {DATA_FILES_URL_BASE}/labels.yaml"""
 
     url = f"/repos/{repo}/labels"
-    repo_labels = {lbl["name"]: lbl for lbl in paginated_get(url, session=get_github_session())}
+    repo_labels = get_repo_labels(repo)
```

Comment:
Nice, so we thought about this in one part of the code.

Reply:
Finally, an up-side to "two ways for everything": one of the ways was right :)